test/test_env_reqs.py

```diff
@@ -0,0 +1,9 @@
+import requirements.txt
+import test_requirements.txt
```

Comment:
You need to import the libraries rather than these text files

Reply:
how do you do that

Reply:
```python
import numpy
import tensorflow
```